### PR TITLE
add context property to check response message

### PR DESCRIPTION
**Pull request by gertd**
See:
https://github.com/aserto-dev/pb-directory/pull/84
https://github.com/aserto-dev/go-directory/pull/54
https://github.com/aserto-dev/azm/pull/55
https://github.com/aserto-dev/go-edge-ds/pull/101

### CODE REVIEW

File: pkg/cli/cmd/templates/install.go [hunk 1]
```diff
@@ -353,7 +353,6 @@ func (i *tmplInstaller) runTemplateTests() error {
 	for _, v := range tests {
 		command := directory.TestExecCmd{
 			File:    v,
-			NoColor: false,
```
Comment (ronenh) [inline pkg/cli/cmd/templates/install.go:356]: Doesn't this mean that `topaz templates install` would only run the directory tests?
Don't we need to run `authorizer.TestExecCmd` too?

Comment (gertd) [reply pkg/cli/cmd/templates/install.go:356]: Yes, will add

File: pkg/cli/cmd/directory/test.go [hunk 2]
```diff
@@ -2,49 +2,39 @@ package directory
...
-	errored         string = "ERR "
-)
-
 type TestCmd struct {
 	Exec     TestExecCmd     `cmd:"" help:"execute assertions"`
 	Template TestTemplateCmd `cmd:"" help:"output assertions template"`
 }
 
 type TestExecCmd struct {
 	File    string `arg:""  default:"assertions.json" help:"filepath to assertions file"`
```
Comment (ronenh) [inline pkg/cli/cmd/directory/test.go:28]: We can do it in a separate PR but I think it would be nice to allow multiple files to be specified. 
```go
Files  []string `arg:"" default:"assertions.json" sep:"none" help:"path to assertions file"`
```
That would allow users to use shell expansions and run a set of test files in a single call:
```shell
topaz ds test exec test/*.json
```

And also do things like:
```shell
cat tests.txt | xargs topaz ds test exec
```
